assets/built-in/transformers/dockerfilegenerator/nodejs/templates/Dockerfile

```diff
@@ -12,17 +12,7 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
-RUN microdnf update && microdnf install wget xz tar && microdnf clean all && \
-    {{- if .NodeVersionProperties.url }}
-    wget {{ .NodeVersionProperties.url }} && \
-    {{- else }}
-    wget https://nodejs.org/dist/{{ .NodeVersion }}/node-{{ .NodeVersion }}-linux-x64.tar.xz && \
-    {{- end }}
-    tar -xJf node-{{ .NodeVersion }}-linux-x64.tar.xz && \
-    mv node-{{ .NodeVersion }}-linux-x64 /node && \
-    rm -f node-{{ .NodeVersion }}-linux-x64.tar.xz
-ENV PATH="$PATH:/node/bin"
+FROM registry.access.redhat.com/ubi8/nodejs-{{ .NodeMajorVersion }}
 COPY . .
 RUN {{ .PackageManager }} install
```

Comment:
Does the image have both npm and yarn?

Reply:
The ubi8/nodejs-{version} base image only has npm and doesn't have yarn. Earlier in the ubi8-minimal image, we were downloading node but not installing yarn. So, I have added the command to install yarn if the PackageManager is yarn. https://github.com/Akash-Nayak/move2kube/blob/03df5e77bd6734eb18dfd6645b3c108549655203/assets/built-in/transformers/dockerfilegenerator/nodejs/templates/Dockerfile#L18